keypom/keypom-airfoil
# Gallery

# Description
Currently this uses the dummy data with a similar chakraUI setup as the other one, putting the links to the gallery and secondary gallery on the new navbar and the new pages and dynamic routing in the router. There are still issues with the images in the events and gallery pages


# How to test

1. check out locally, and go to http://localhost:3000
2. check out the new nav links

# Screenshots/Screen Recording of Implementation
Visuals are WIP

![image](https://github.com/keypom/keypom-frontend/assets/3915233/fcfe67c5-20fb-4888-addd-c3f988184f56)
![image](https://github.com/keypom/keypom-frontend/assets/3915233/b9315b2d-334f-43bc-b69a-19b2587740a8)
